app/src/main/res/navigation/main_nav.xml

```diff
@@ -94,14 +104,64 @@
         android:label="TransactionDetailsFragment"
         tools:layout="@layout/fragment_transaction_details">
 
+        <action
+            android:id="@+id/action_transactionDetailsFragment_to_advancedTransactionDetailsFragment"
+            app:destination="@id/advancedTransactionDetailsFragment"
+            app:popUpTo="@id/transactionDetailsFragment" />
+
         <argument
             android:name="txId"
             app:argType="string" />
 
+    </fragment>
+
+    <fragment
+        android:id="@+id/creationTransactionDetailsFragment"
+        android:name="io.gnosis.safe.ui.transactions.details.CreationTransactionDetailsFragment"
+        android:label="CreationTransactionDetailsFragment"
+        tools:layout="@layout/fragment_transaction_details_creation">
+
         <action
```

Comment:
why is this action inside tx details fragment?

Reply:
Removed in 4a948ac8ff2a0696ec41e88674941c788f6cb3eb